zaza/openstack/charm_tests/ceph/mon/tests.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Ceph-mon Testing for cinder-ceph."""
+
+import logging
+
+import zaza.model
+
+from zaza.openstack.utilities import (
+    generic as generic_utils,
+    openstack as openstack_utils,
+    exceptions as zaza_exceptions
+)
+import zaza.openstack.charm_tests.test_utils as test_utils
+
+
+class CinderCephMonTest(test_utils.OpenStackBaseTest):
+    """Verify that the ceph mon units are healthy."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for running ceph mon tests with cinder."""
+        super().setUpClass()
```

Comment:
This seems a bit boiler-plate - do we really need to include it if all we're doing is deferring to `super`?

Reply:
We probably don't, but I tend to leave them in as a reminder that there is that entry point. It can be dropped though.

Reply:
Fair enough